src/main/java/seedu/address/logic/parser/DeleteModuleParser.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.logic.commands.DeleteModuleCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+//@@author GabrielYik
+
+/**
+ *
+ */
+public class DeleteModuleParser implements Parser<DeleteModuleCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the DeleteModuleCommand
+     * and returns a DeleteModuleCommand object for execution.
+     *
+     * @throws ParseException if the user input does not conform to the expected format
+     */
+    @Override
+    public DeleteModuleCommand parse(String args) throws ParseException {
+        try {
+            String code = ParserUtil.parseCode(args);
```

Comment:
It is not immediately clear that `ParserUtil.parseCode` is for parsing module code. How about renaming as `ParserUtil.parseModuleCode`.